integration_tests/pages/recordAttendance/activitiesPage.ts

```diff
@@ -13,5 +13,11 @@ export default class ActivitiesPage extends Page {
         return Cypress.$.makeArray($el)
       })
 
-  selectActivityWithName = (activityName: string) => this.activityRows().find(`a:contains(${activityName})`).click()
+  selectActivityWithName = (activityName: string) =>
+    this.activityRows()
+      .find(`a:contains(${activityName})`)
+      .then(e => {
+        const activityUrl = e.prop('href')
+        cy.visit(activityUrl)
+      })
 }
```

Comment:
Cypress doesn't support new tabs, so instead of clicking this link we extract the URL and navigate to the page.